script/Arcueid/Day 4/04_00_ARC04_4_1B.txt

```diff
@@ -273,7 +273,8 @@ Jumping to get in its line of fire, the woman in white flicks the lightning away
 -- Page 277, Offset 6327.
 -- ……恐ろしいのは、女の跳躍があまりにも軽やかだった事だ。あの黒豹の速度を上回りながら、なお“軽やか”に見える事が、俺の神経を逆なでする。
 What's frightening is... her leap looked so effortless.
- The fact that she moved even faster than that black panther yet still made it seem easy really rubs me the wrong way.
+ The fact that she moved even faster than that black panther,
+ yet still made it seem easy really rubs me the wrong way.
```

Comment:
would this be fine in game without a leading space?

Reply:
Nothing really looks fine without a leading space because then deepLuna doesn't know how to break the lines. It uses spaces to know where word boundaries are